muse_for_anything/db/models/object_relation_tables.py

```diff
@@ -31,12 +31,12 @@ class OntologyTypeVersionToTypeVersion(MODEL, IdMixin):
 
     # the type version importing
     type_version_source_id: Mapped[int] = mapped_column(
-        ForeignKey("OntologyObjectTypeVersion.id"), nullable=False
+        ForeignKey(OntologyObjectTypeVersion.id), nullable=False
```

Comment:
Should be ok, these stringified expressions are sometimes required to break up circular dependencies...